client/components/vertical-nav/index.jsx

```diff
@@ -2,15 +2,17 @@
  * External dependencies
  */
 import React from 'react';
+import style from './style';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
 
 const VerticalNav = React.createClass( {
 	render() {
 		return (
-			<div className="vertical-nav">
+			<div className={ style.verticalNav }>
```

Comment:
This uses the stylesheet that is already there for vertical-nav - that's one of the advantages of this approach - we don't need to change any of the CSS files to convert components.